### PR TITLE
pganalyze Blog and Documentation

### PR DESCRIPTION
**Pull request by ianstanton**
Blog and documentation for pganalyze integration,

### CODE REVIEW

File: src/content/blog/2024-04-02-pganalyze/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+pganalyze dashboard under the “Settings” section. Copy your API key to use in the next step.
+
+![api-key.png](api-key.png)
+
+## Step 3: Activate the pganalyze App in Tembo Cloud
+To connect your Tembo Cloud instance to pganalyze, you will need to activate the pganalyze App for your instance.
+
+1. Log in to Tembo Cloud at [cloud.tembo.io](https://cloud.tembo.io/).
+2. Select the instance you want to connect to pganalyze.
+3. Click on the “Apps” tab in the instance details page.
```
Comment (samay-sharma) [inline src/content/blog/2024-04-02-pganalyze/index.md:54]: We should have a screenshot here.

We also have a step missing for clicking on the pganalyze app.

File: src/content/blog/2024-04-02-pganalyze/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+4. Submit your pganalyze API key in the “API Key” section and click “Activate”.
+
+Once activated, pganalyze will start collecting and analyzing your database performance data. You can access your pganalyze
+dashboard by clicking on the “pganalyze app” button.
+
+pganalyze will automatically start collecting and analyzing your database performance data, and provide you
+with detailed insights into your database performance.
+
+# Tuning Advisors
```
Comment (samay-sharma) [inline src/content/blog/2024-04-02-pganalyze/index.md:60]: In the sub-sections of this, can you add relevant links to different pganalyze docs?

File: src/content/blog/2024-04-02-pganalyze/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+date: 2024-04-02T12:00
+description: Announcing support for pganalyze on Tembo Cloud
+---
+
+At Tembo, we’ve been building a world-class, developer friendly managed Postgres solution. We’re excited to announce
+Tembo Cloud now integrates seamlessly with pganalyze - a powerful tool that helps you understand and optimize your
+Postgres performance. With this integration, you can now easily monitor, analyze your Tembo Cloud Postgres instance with
+detailed insights into your database performance and configure alerts to receive notifications about the same.
+
+# What is pganalyze?
```
Comment (samay-sharma) [inline src/content/blog/2024-04-02-pganalyze/index.md:19]: Can we add a screenshot here?

File: src/content/blog/2024-04-02-pganalyze/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
+slug: pganalyze
+title: 'Advanced PostgreSQL metrics and insights with pganalyze on Tembo Cloud'
```
Comment (samay-sharma) [inline src/content/blog/2024-04-02-pganalyze/index.md:3]: For some reason I prefer "Advanced PostgreSQL metrics and insights on Tembo Cloud with pganalyze"

**Review by samay-sharma — COMMENTED** [review]
Overall, looks good. Pushed a small commit with some minor edits.

Two main things to do a pass with:
* Add more screenshots where relevant.
* Add more links wherever possible.

File: src/content/blog/2024-04-02-pganalyze/index.md (hunk 1 above)
Comment (ianstanton) [reply src/content/blog/2024-04-02-pganalyze/index.md:56]: Added temporary screenshot while @vrmiguel is working on finishing touches for the `pganalyze App` button

Comment (ianstanton) [reply src/content/blog/2024-04-02-pganalyze/index.md:56]: Updated with latest

File: src/content/blog/2024-04-02-pganalyze/index.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+## Step 3: Activate the pganalyze App in Tembo Cloud
+To connect your Tembo Cloud instance to pganalyze, you will need to activate the pganalyze App for your instance.
+
+![app.png](app.png)
+
+1. Log in to Tembo Cloud at [cloud.tembo.io](https://cloud.tembo.io/).
+2. Select the instance you want to connect to pganalyze.
+3. Click on the “Apps” tab in the instance details page.
+4. Submit your pganalyze API key in the “API Key” section and click “Activate”.
```
Comment (samay-sharma) [inline src/content/blog/2024-04-02-pganalyze/index.md:59]: Shouldn't there be another step which is clicking on the pganalyze app?

File: src/content/blog/2024-04-02-pganalyze/index.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+pganalyze provides detailed insights into your database logs, including slow queries, errors, and other important events.
+To make this possible on Tembo Cloud, we've made some contributions to the open source [pganalyze collector](https://github.com/pganalyze/collector/).
+This allows pganalyze to collect logs from your Tembo Cloud instance and provide you with detailed insights into your database logs.
+
+![log-insights.png](log-insights.png)
+
+With Log Insights enabled, we can make use of other powerful features, such as EXPLAIN Plans.
+
+# EXPLAIN Plans
+pganalyze simplifies viewing EXPLAIN plans by integrating with the auto_explain extension. It highlights crucial
```
Comment (samay-sharma) [inline src/content/blog/2024-04-02-pganalyze/index.md:97]: Link to auto_explain in trunk

File: src/content/blog/2024-04-02-pganalyze/index.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+information in query plans, making important data more noticeable. To make this integration as seamless as possible, we
+automatically modify configs and extensions in your Tembo instance. More details about this feature can be found in the 
+[pganalyze documentation](https://pganalyze.com/docs/explain).
+
+![explain-plans.png](explain-plans.png)
+
+# Alerts and Check-Up
+pganalyze also provides powerful alerting and notification features that help you stay on top of your database performance.
```
Comment (samay-sharma) [inline src/content/blog/2024-04-02-pganalyze/index.md:105]: inline link alerts and notification docs for pganalyze

**Review by samay-sharma — APPROVED** [review]
Made 3 minor comments. Other than that, I think good to go.

File: src/content/blog/2024-04-02-pganalyze/index.md (hunk 5 above)
Comment (ianstanton) [reply src/content/blog/2024-04-02-pganalyze/index.md:59]: There are two occasions where the user needs to navigate to the pganalyze app:
1. When obtaining their API Key. This is addressed in steps 1 (Sign up for pganalyze) and 2 (Obtain your pganalyze API Key)
2. After they've activated the app. This is addressed in step 5 of this section
  > 5.  Click the “pganalyze App” button to access your pganalyze dashboard.

File: src/content/blog/2024-04-02-pganalyze/index.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+## Step 2: Obtain your pganalyze API Key
+Once you have signed up for pganalyze, you will need to obtain your pganalyze API key. You can find your API key in the
+pganalyze dashboard under the “Settings” section. Copy your API key to use in the next step.
+
+![api-key.png](api-key.png)
+
+## Step 3: Activate the pganalyze App in Tembo Cloud
+To connect your Tembo Cloud instance to pganalyze, you will need to activate the pganalyze App for your instance.
+
+![app.png](app.png)
```
Comment (ianstanton) [inline src/content/blog/2024-04-02-pganalyze/index.md:54]: Update image

**Pull request merged**